src/Resources/app/administration/src/component/adyen-config-check-button/adyen-config-check-button.html.twig

```diff
@@ -1,8 +1,9 @@
-<div>
+<template>
     <sw-button-process
+            :disabled="isLoading"
             :isLoading="isLoading"
             :processSuccess="isSaveSuccessful"
             @process-finish="saveFinish"
             @click="check"
-    >{{ label }}</sw-button-process>
-</div>
+    >Test Configuration</sw-button-process>
```

Comment:
From v6.3.4.1 the {{label}} is no longer being displayed on the button in the Admin, moving the text from the config.xml to the twig template works across all the versions.

Reply:
Nice. Can it also be replaced with a snippet so it can be translated? https://github.com/Adyen/adyen-shopware6/tree/develop/src/Resources/snippet

Reply:
Now that I think about it the whole form is not translated anyway. So maybe we do the whole translation as a separated low prio ticket.

Reply:
yeah that was my thinking as well